src/ServiceInsight/Saga/SagaResources.xaml

```diff
@@ -25,4 +25,13 @@
     
     <SolidColorBrush x:Key="FailedMessageBackground" Color="#F9916F" />
     <SolidColorBrush x:Key="RetriedMessageBackground" Color="#F2F2F2" />
+
+    <Style x:Key="GrayNullTextStyle" TargetType="{x:Type TextBlock}">
```

Comment:
This style doesn't make the text gray when it's null. Should the name be changed? Something more generic like `ValueTextStyle`